gobblin-data-management/src/main/java/gobblin/data/management/copy/converter/DecryptConverter.java

```diff
@@ -66,4 +71,12 @@ public String convertSchema(String inputSchema, WorkUnitState workUnit) throws S
     }
   }
 
+  /**
+   * Remove {@value #GPG_EXTENSION} from {@link CopyableFile#getDestination()} and
+   * {@link CopyableFile#getRelativeDestination()}
+   */
+  private void removeExtension(CopyableFile file) {
+    file.setDestination(new Path(file.getDestination().toString().replace(GPG_EXTENSION, "")));
```

Comment:
You probably can use http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/io/Files.html#getNameWithoutExtension(java.lang.String).